### PR TITLE
Fix healthcheck to look for entry of all IPv4 addresses (0.0.0.0)

### LINKED ISSUE #7993: Docker service healthcheck fails every 4 or 5 minutes
I have a docker stack of 6 minio services distributed between 3 hosts. I followed the [documentation](https://docs.min.io/docs/deploy-minio-on-docker-swarm).

But my minio services keep restarting every 4 or 5 minutes. 'docker inspect' reveals a unhealthy status

## Expected Behavior
Service healthcheck should work.

## Current Behavior
Every 4 or 5 minutes, healthcheck fails for random services.

```
# date && docker stack ps minio
Tue Jul 30 09:32:06 CEST 2019
ID                  NAME                IMAGE                                      NODE                            DESIRED STATE       CURRENT STATE            ERROR               PORTS
3aibaudl4g9n        minio_minio3.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva315.gva.dts.corp.local   Running             Running 5 minutes ago
diqd5s0q1glf        minio_minio2.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva313.gva.dts.corp.local   Running             Running 5 minutes ago
bm7gt51plzwc        minio_minio1.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva313.gva.dts.corp.local   Running             Running 5 minutes ago
g38fkhc0wcw6        minio_minio6.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva321.gva.dts.corp.local   Running             Running 6 minutes ago
qlzc5mecgwfj        minio_minio5.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva321.gva.dts.corp.local   Running             Running 6 minutes ago
mmv9kqv3vw84        minio_minio4.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva315.gva.dts.corp.local   Running             Running 6 minutes ago

# date && docker stack ps minio
Tue Jul 30 09:32:30 CEST 2019
ID                  NAME                IMAGE                                      NODE                            DESIRED STATE       CURRENT STATE             ERROR               PORTS
01emgeljpxyt        minio_minio2.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva313.gva.dts.corp.local   Running             Starting 3 seconds ago
r5kgoivji2cv        minio_minio1.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva313.gva.dts.corp.local   Running             Starting 7 seconds ago
3aibaudl4g9n        minio_minio3.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva315.gva.dts.corp.local   Running             Running 6 minutes ago
g38fkhc0wcw6        minio_minio6.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva321.gva.dts.corp.local   Running             Running 6 minutes ago
qlzc5mecgwfj        minio_minio5.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva321.gva.dts.corp.local   Running             Running 6 minutes ago
mmv9kqv3vw84        minio_minio4.1      minio/minio:RELEASE.2019-07-05T21-20-21Z   lnx-gva315.gva.dts.corp.local   Running             Running 6 minutes ago
```

Stopped service logs are:

```
# docker logs 8436b645411e
Waiting for a minimum of 3 disks to come online (elapsed 1s)
Waiting for a minimum of 3 disks to come online (elapsed 2s)
Waiting for a minimum of 3 disks to come online (elapsed 3s)
Waiting for a minimum of 3 disks to come online (elapsed 6s)
Waiting for a minimum of 3 disks to come online (elapsed 16s)

Unable to connect to http://minio1:9000/export: Post http://minio1:9000/minio/storage/v6/export/getinstanceid?: dial tcp: lookup minio1 on 127.0.0.11:53: no such host
Unable to connect to http://minio3:9000/export: Post http://minio3:9000/minio/storage/v6/export/getinstanceid?: dial tcp: lookup minio3 on 127.0.0.11:53: no such host
Unable to connect to http://minio5:9000/export: Post http://minio5:9000/minio/storage/v6/export/getinstanceid?: dial tcp: lookup minio5 on 127.0.0.11:53: no such host
Unable to connect to http://minio6:9000/export: Post http://minio6:9000/minio/storage/v6/export/getinstanceid?: dial tcp: lookup minio6 on 127.0.0.11:53: no such host

Waiting for a minimum of 3 disks to come online (elapsed 44s)

Status:         6 Online, 0 Offline.
Endpoint:  http://10.255.0.191:9000  http://10.0.4.42:9000  http://172.18.0.7:9000  http://127.0.0.1:9000

Browser Access:
   http://10.255.0.191:9000  http://10.0.4.42:9000  http://172.18.0.7:9000  http://127.0.0.1:9000

Object API (Amazon S3 compatible):
   Go:         https://docs.min.io/docs/golang-client-quickstart-guide
   Java:       https://docs.min.io/docs/java-client-quickstart-guide
   Python:     https://docs.min.io/docs/python-client-quickstart-guide
   JavaScript: https://docs.min.io/docs/javascript-client-quickstart-guide
   .NET:       https://docs.min.io/docs/dotnet-client-quickstart-guide

Exiting on signal: TERMINATED
```

Sample of the inspect on a stopped service:

```
# docker inspect 8436b645411e
[
    {
        "Id": "8436b645411ece2b73fb5cf81af6b9472e1e273b8a595954ffeb4053f6b8f905",
        "Created": "2019-07-30T07:25:08.717049677Z",
        "Path": "/usr/bin/docker-entrypoint.sh",
(...)
        "State": {
            "Status": "exited",
            "Running": false,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 0,
            "ExitCode": 0,
            "Error": "",
            "StartedAt": "2019-07-30T07:25:13.614747038Z",
            "FinishedAt": "2019-07-30T07:32:17.555973818Z",
            "Health": {
                "Status": "unhealthy",
                "FailingStreak": 3,
                "Log": [
                    {
                        "Start": "2019-07-30T09:28:13.945438907+02:00",
                        "End": "2019-07-30T09:28:14.166277934+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2019-07-30T09:29:14.187686747+02:00",
                        "End": "2019-07-30T09:29:14.346482798+02:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2019-07-30T09:30:14.356339818+02:00",
                        "End": "2019-07-30T09:30:14.642945929+02:00",
                        "ExitCode": 1,
                        "Output": "2019/07/30 07:30:14 no minio process found\n"
                    },
                    {
                        "Start": "2019-07-30T09:31:14.660750391+02:00",
                        "End": "2019-07-30T09:31:14.824644118+02:00",
                        "ExitCode": 1,
                        "Output": "2019/07/30 07:31:14 no minio process found\n"
                    },
                    {
                        "Start": "2019-07-30T09:32:14.838300576+02:00",
                        "End": "2019-07-30T09:32:14.982497632+02:00",
                        "ExitCode": 1,
                        "Output": "2019/07/30 07:32:14 no minio process found\n"
(...)
            "Healthcheck": {
                "Test": [
                    "CMD-SHELL",
                    "healthcheck"
                ],
                "Interval": 60000000000
            },
(...)
```

Full inspect log: [minio-inspect.log](https://github.com/minio/minio/files/3445816/minio-inspect.log)

## Possible Solution
No idea why the healthcheck fails.
But I launched a shell in a running service and manually execute the healthcheck command 'healthcheck'. Many times, the return code was 0 with no output. But then I caught this :

```
/ # healthcheck
2019/07/30 07:54:33 no minio process found
/ # ps aux
PID   USER     TIME  COMMAND
    1 root      0:00 minio server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
   65 root      0:00 sh
  130 root      0:00 ps aux
```

I guess the healthcheck command should have found the minio process ?

Service state is fine if I disable the healthcheck with:
```
    healthcheck:
      disable: true
```
It can run several hours without any issue.

## Steps to Reproduce (for bugs)

Here is the compose file:

My compose file (without the healthcheck: disable: true)

```
version: "3.7"

services:
  minio1:
    image: minio/minio:RELEASE.2019-07-05T21-20-21Z
    hostname: minio1
    volumes:
      - /opt/minio1-data:/export
    ports:
      - "9001:9000"
    deploy:
      restart_policy:
        delay: 10s
        window: 60s
      placement:
        constraints:
          - node.labels.minioserver1==true
      labels:
        - "traefik.frontend.rule=Host: minio.dts.corp.local"
        - "traefik.enable=true"
        - "traefik.port=9000"
        - "traefik.docker.network=minio_default"
    command: server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
    secrets:
      - secret_key
      - access_key

  minio2:
    image: minio/minio:RELEASE.2019-07-05T21-20-21Z
    hostname: minio2
    volumes:
      - /opt/minio2-data:/export
    ports:
      - "9002:9000"
    deploy:
      restart_policy:
        delay: 10s
        window: 60s
      placement:
        constraints:
          - node.labels.minioserver1==true
      labels:
        - "traefik.frontend.rule=Host: minio.dts.corp.local"
        - "traefik.enable=true"
        - "traefik.port=9000"
        - "traefik.docker.network=minio_default"
    command: server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
    secrets:
      - secret_key
      - access_key

  minio3:
    image: minio/minio:RELEASE.2019-07-05T21-20-21Z
    hostname: minio3
    volumes:
      - /opt/minio3-data:/export
    ports:
      - "9003:9000"
    deploy:
      restart_policy:
        delay: 10s
        window: 60s
      placement:
        constraints:
          - node.labels.minioserver2==true
      labels:
        - "traefik.frontend.rule=Host: minio.dts.corp.local"
        - "traefik.enable=true"
        - "traefik.port=9000"
        - "traefik.docker.network=minio_default"
    command: server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
    secrets:
      - secret_key
      - access_key

  minio4:
    image: minio/minio:RELEASE.2019-07-05T21-20-21Z
    hostname: minio4
    volumes:
      - /opt/minio4-data:/export
    ports:
      - "9004:9000"
    deploy:
      restart_policy:
        delay: 10s
        window: 60s
      placement:
        constraints:
          - node.labels.minioserver2==true
      labels:
        - "traefik.frontend.rule=Host: minio.dts.corp.local"
        - "traefik.enable=true"
        - "traefik.port=9000"
        - "traefik.docker.network=minio_default"
    command: server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
    secrets:
      - secret_key
      - access_key

  minio5:
    image: minio/minio:RELEASE.2019-07-05T21-20-21Z
    hostname: minio5
    volumes:
      - /opt/minio5-data:/export
    ports:
      - "9005:9000"
    deploy:
      restart_policy:
        delay: 10s
        window: 60s
      placement:
        constraints:
          - node.labels.minioserver3==true
      labels:
        - "traefik.frontend.rule=Host: minio.dts.corp.local"
        - "traefik.enable=true"
        - "traefik.port=9000"
        - "traefik.docker.network=minio_default"
    command: server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
    secrets:
      - secret_key
      - access_key

  minio6:
    image: minio/minio:RELEASE.2019-07-05T21-20-21Z
    hostname: minio6
    volumes:
      - /opt/minio6-data:/export
    ports:
      - "9006:9000"
    deploy:
      restart_policy:
        delay: 10s
        window: 60s
      placement:
        constraints:
          - node.labels.minioserver3==true
      labels:
        - "traefik.frontend.rule=Host: minio.dts.corp.local"
        - "traefik.enable=true"
        - "traefik.port=9000"
        - "traefik.docker.network=minio_default"
    command: server http://minio1/export http://minio2/export http://minio3/export http://minio4/export http://minio5/export http://minio6/export
    secrets:
      - secret_key
      - access_key

secrets:
  secret_key:
    external: true
  access_key:
    external: true
```

## Regression
Before switching on our production environment under CentOS and Docker 19.03, all my previous tests runned fine on VMWare PhotonOS 1.0 with Docker 18.03 and Minio RELEASE.2019-04-04T18-31-46Z.

## Your Environment
Minio version: RELEASE.2019-07-05T21-20-21Z and RELEASE.2019-07-24T02-02-23Z fails both.
Docker version: Docker CE 19.03.1
Docker Host OS : CentOS Linux release 7.6.1810 (Core)

### PR DESCRIPTION
**Pull request by nitisht**
## Description
For containers not supporting IPv6 interface, netstat responds with `0.0.0.0`
as the local address. We should check for this address as well when looking for 
MinIO processes.

```
/ # netstat -ntlp
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:9000            0.0.0.0:*               LISTEN      1/minio
tcp        0      0 127.0.0.11:43952        0.0.0.0:*               LISTEN      -
```

## Motivation and Context
Fixes #7993

## How to test this PR?
- Start MinIO server container without IPv6 support
- Look for healthcheck command output, it should return `0`

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

File: dockerscripts/healthcheck.go [hunk 1]
```diff
@@ -71,14 +72,17 @@ func findEndpoint() (string, error) {
...
+	// (on ::: or 0.0.0.0) inside container.
+	// Since MinIO may run as non-root user, we can
 	// not depend on the PID/Program name column
 	// of netstat output
 	for scanner.Scan() {
 		line := scanner.Text()
-		if strings.Contains(line, tcp) && strings.Contains(line, anyIP) {
-			newLine := strings.Replace(line, anyIP, "127.0.0.1:", 1)
+		if strings.Contains(line, tcp) && (strings.Contains(line, anyIPv6) || strings.Contains(line, anyIPv4)) {
+			newLine := strings.Replace(line, anyIPv4, "127.0.0.1:", 1)
```
Comment (kannappanr) [inline dockerscripts/healthcheck.go:82]: Is it still `127.0.0.1` even for IPv6

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#343cbc2f-83f1-48b6-ac8b-5512b9042423) |
| mint-large-bucket.sh | :x: [more...](#834b2312-6ed9-43a9-80b3-5f4aacc3c11d) |
<a name="834b2312-6ed9-43a9-80b3-5f4aacc3c11d"></a>
### 8046-f005048/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30930
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e76efb912155:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 55 seconds
(4/14) Running aws-sdk-ruby tests ... done in 14 seconds
(5/14) Running awscli tests ... done in 1 minutes and 49 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 40 seconds
{
  "name": "mc",
  "duration": "118192",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-23531 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-447  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-447/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-447/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-447/datafile-1-kB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-447/datafile-1-b`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-447/datafile-10-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-447/datafile-1.03-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-447/datafile-100-kB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-447/datafile-10-kB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-447/datafile-129-MB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-447/datafile-11-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-447/datafile-33-kB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-447/datafile-5-MB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-447/datafile-6-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-447/datafile-5243880-b`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-447/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 496.67 MiB, Speed: 4.20 MiB/s"
}

Executed 6 out of 14 tests successfully.
```
<a name="343cbc2f-83f1-48b6-ac8b-5512b9042423"></a>
### 8046-f005048/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31595
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp dd734b679767:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 13 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-slpjew0kk4hp9e",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 12545,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31595/aws-sdk-go-test-slpjew0kk4hp9e: dial tcp 72.28.97.57:31595: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

File: dockerscripts/healthcheck.go (hunk 1 above)
Comment (harshavardhana) [reply dockerscripts/healthcheck.go:82]: Yes it looks little confusing.. can you shed some light @nitisht

Comment (nitisht) [reply dockerscripts/healthcheck.go:82]: You're right @kannappanr if it is only IPv6, https://github.com/minio/minio/pull/8046/files#diff-e354ae019f16b5ff940a9e155ae1ee26R84 should use `::1`, will make the change

**Review by kannappanr — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#43ed9eb9-ce20-4bc3-9f5f-643742aeda77) |
| mint-large-bucket.sh | :x: [more...](#2f111f99-48bd-4129-a374-30104e203c9d) |
| mint-compression-fs.sh | :x: [more...](#809660b0-67ae-4800-887b-c2d603f30be6) |
<a name="2f111f99-48bd-4129-a374-30104e203c9d"></a>
### 8046-0711705/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32166
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 97ce0dd4eb90:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 15 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-oqftauyz2mkklr",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 15055,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32166/aws-sdk-go-test-oqftauyz2mkklr: dial tcp 72.28.97.57:32166: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="43ed9eb9-ce20-4bc3-9f5f-643742aeda77"></a>
### 8046-0711705/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31426
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5f68157b9bff:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 19 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-y3lzicsw1st6sq",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 18372,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31426/aws-sdk-go-test-y3lzicsw1st6sq: dial tcp 72.28.97.57:31426: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="809660b0-67ae-4800-887b-c2d603f30be6"></a>
### 8046-0711705/mint-compression-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32110
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 00c98173ea9d:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 16 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-d19vmksbwtu24n",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 15342,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32110/aws-sdk-go-test-d19vmksbwtu24n: dial tcp 72.28.97.57:32110: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**